flow/core/rewards.py

```diff
@@ -469,24 +468,10 @@ def instantaneous_power(env, veh_ids=None, gain=.001):
         veh_ids = [veh_ids]
     for veh_id in veh_ids:
         speed = env.k.vehicle.get_speed(veh_id)
-        prev_speed = env.k.vehicle.get_previous_speed(veh_id)
-        # negative acceleration indicates regenerative breaking for some energy models
-        accel = (speed - prev_speed) / env.sim_step
+        accel = env.k.vehicle.get_accel_no_noise_with_failsafe(veh_id)
         grade = 0
```

Comment:
you can also use `env.k.vehicle.get_road_grade(veh_id)` here, which currently returns 0. this way, this will automatically work when that method is filled in.